### PR TITLE
e2e: fix memory manager tests

### LINKED ISSUE #99070: Memory manager e2e tests fail
#### Which jobs are failing:
e2e-memory-manager

#### Which test(s) are failing:
Different tests under the memory manager lane

#### Testgrid link:
https://testgrid.k8s.io/sig-node-kubelet#kubelet-serial-gce-e2e-memory-manager

#### Reason for failure:
The node does not have configured hugepages
```
"Event occurred" object="memory-manager-test-9264/memory-manager-none78fqv" kind="Pod" apiVersion="v1" type="Warning" reason="OutOfhugepages-2Mi" message="Node didn't have enough resource: hugepages-2Mi, requested: 134217728, used: 0, capacity: 0"
```

### PR DESCRIPTION
**Pull request by cynepco3hahue**
#### What type of PR is this?

/kind bug
/kind failing-test

#### What this PR does / why we need it:
Restart the kubelet and wait for hugepages resources to appear under the node.

#### Which issue(s) this PR fixes:

Fixes #99070

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: test/e2e_node/memory_manager_test.go [hunk 1]
```diff
@@ -305,8 +305,42 @@ var _ = SIGDescribe("Memory Manager [Serial] [Feature:MemoryManager][NodeAlphaFe
...
+					return fmt.Errorf("the node does not have the resource %s", hugepagesResourceName2Mi)
+				}
+
+				size, succeed := capacity.AsInt64()
+				if !succeed {
+					return fmt.Errorf("failed to convert quantity to int64")
+				}
+
+				// 256 Mb
+				expectedSize := int64(hugepagesCount * hugepagesSize2M * 1024)
```
Comment (swatisehgal) [inline test/e2e_node/memory_manager_test.go:338]: Can we add a comment to make it more clear that the expected size is in bytes?

**Review by swatisehgal — COMMENTED** [review]
Just a minor suggestion but other than that it looks good to me!

File: test/e2e_node/memory_manager_test.go (hunk 1 above)
Comment (cynepco3hahue) [reply test/e2e_node/memory_manager_test.go:338]: done

**Comment by ehashman** [comment]
/milestone v1.21

Discussed at 2021-03-17 Node CI meeting for test freeze.

**Review by mrunalp — APPROVED** [review]

**Pull request merged**